osu.Game/Skinning/Components/BeatmapAttributeText.cs

```diff
@@ -0,0 +1,133 @@
+            [BeatmapAttribute.Title] = EditorSetupStrings.Title,
+            [BeatmapAttribute.Artist] = EditorSetupStrings.Artist,
+            [BeatmapAttribute.DifficultyName] = EditorSetupStrings.DifficultyHeader,
+            [BeatmapAttribute.Creator] = EditorSetupStrings.Creator,
+            [BeatmapAttribute.Length] = ArtistStrings.TracklistLength.ToTitle(),
+            [BeatmapAttribute.RankedStatus] = BeatmapDiscussionsStrings.IndexFormBeatmapsetStatusDefault,
+            [BeatmapAttribute.BPM] = BeatmapsetsStrings.ShowStatsBpm,
+        }.ToImmutableDictionary();
+
+        private readonly OsuSpriteText text;
+
+        public BeatmapAttributeText()
+        {
+            AutoSizeAxes = Axes.Both;
+
+            InternalChildren = new Drawable[]
+            {
+                text = new OsuSpriteText
+                {
+                    Anchor = Anchor.CentreLeft,
+                    Origin = Anchor.CentreLeft,
+                    Font = OsuFont.Default.With(size: 40)
+                }
+            };
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            Attribute.BindValueChanged(_ => updateLabel());
+            Template.BindValueChanged(_ => updateLabel());
+            beatmap.BindValueChanged(b =>
+            {
+                updateBeatmapContent(b.NewValue);
+                updateLabel();
+            }, true);
+        }
+
+        private void updateBeatmapContent(WorkingBeatmap workingBeatmap)
+        {
+            valueDictionary[BeatmapAttribute.Title] = workingBeatmap.BeatmapInfo.Metadata.Title;
+            valueDictionary[BeatmapAttribute.Artist] = workingBeatmap.BeatmapInfo.Metadata.Artist;
+            valueDictionary[BeatmapAttribute.DifficultyName] = workingBeatmap.BeatmapInfo.DifficultyName;
+            valueDictionary[BeatmapAttribute.Creator] = workingBeatmap.BeatmapInfo.Metadata.Author.Username;
+            valueDictionary[BeatmapAttribute.Length] = TimeSpan.FromMilliseconds(workingBeatmap.BeatmapInfo.Length).ToFormattedDuration();
+            valueDictionary[BeatmapAttribute.RankedStatus] = workingBeatmap.BeatmapInfo.Status.GetLocalisableDescription();
+            valueDictionary[BeatmapAttribute.BPM] = workingBeatmap.BeatmapInfo.BPM.ToString(@"F2");
+            valueDictionary[BeatmapAttribute.CircleSize] = ((double)workingBeatmap.BeatmapInfo.Difficulty.CircleSize).ToString(@"F2");
+            valueDictionary[BeatmapAttribute.HPDrain] = ((double)workingBeatmap.BeatmapInfo.Difficulty.DrainRate).ToString(@"F2");
+            valueDictionary[BeatmapAttribute.Accuracy] = ((double)workingBeatmap.BeatmapInfo.Difficulty.OverallDifficulty).ToString(@"F2");
+            valueDictionary[BeatmapAttribute.ApproachRate] = ((double)workingBeatmap.BeatmapInfo.Difficulty.ApproachRate).ToString(@"F2");
+            valueDictionary[BeatmapAttribute.StarRating] = workingBeatmap.BeatmapInfo.StarRating.ToString(@"F2");
```

Comment:
Similarly, was the intent here to call `.ToString()` rather than `.ToLocalisableString()`?